apps/earn-protocol/app/page.tsx

```diff
@@ -11,14 +13,20 @@ const Form = dynamic(() => import('@/components/organisms/Form/Form'), {
   ssr: false,
 })
 
+// TODO: Replace with the real dynamic values from the UI state later
```

Comment:
We should be able to retrieve the list of fleets from either on-chain or a config file, and return it to the SDK client, so you don't have to hardcode it here. Just as a future task

Reply:
Yes indeed, this is a comment for web app state management